### PR TITLE
/J command

### PR DESCRIPTION
**Pull request by diddledan**
This is very frivolous, but I often use /j as a quick shortcut of /join, but until now this wasn't supported by shout, so I've added the command.

### CODE REVIEW

**Comment by diddledan** [comment]
oh, fudge, this has my other branch's changes :-(

**Comment by astorije** [comment]
Hey @diddledan, thanks a lot for your contribution!
However, this seems to me like a personal preference. What you can do at the moment is type `/j`, hit <kbd>Tab</kbd> and Shout will auto-complete to `/join`. It's a tab away from what you want, but I believe that's really a minor cost (compared to typing `/join` as a whole).

The other reason why I am reluctant to this is that it opens doors to all sorts of aliases and I don't think we want that.
Instead, I would love to see #554 evolve and ship, as well as a dedicated plugin for complexe aliases, and that would be a terrific plugin to make :-)

So in short I am :-1: on this, hoping the plugin system will soon lead somewhere. If you are willing to help on the plugin system or such potential plugin (for example, identifying the contract/event/API exposed by Shout and used by this plugin), that would be greatly appreciated!
I might be :-1: but I'm no unique authority, so I would like another voice from @JocelynDelalande, @erming or @floogulinc before turning this down.

File: client/js/shout.js [hunk 1]
```diff
@@ -366,7 +366,7 @@ $(function() {
 			}
 		);
 		if ([
-			"join",
+			"join","j",
```
Comment (floogulinc) [inline client/js/shout.js]: This is wrong. It seems you just added `"j"` wherever you saw `"join"`. In this part, it's for the settings that hide certain messages in the chat feed.

File: client/js/shout.js [hunk 2]
```diff
@@ -603,7 +603,7 @@ $(function() {
 		}
 
 		var ignore = [
-			"join",
+			"join","j",
```
Comment (floogulinc) [inline client/js/shout.js]: Same thing here as in my other comment.

**Comment by floogulinc** [comment]
We already added this in Shuo-IRC/Shuo#30 along with some other useful aliases. If desired, I'll port this to a PR in Shout.

File: client/js/shout.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ $(function() {
 		"/devoice",
 		"/disconnect",
 		"/invite",
-		"/join",
+		"/join","/j",
```
Comment (floogulinc) [inline client/js/shout.js]: It shouldn't be added to the command list since it is used to autocomplete and typing `/j` and hitting tab should just autocomplete to `/join`

**Comment by floogulinc** [comment]
Also, please refer to issue #364 for the other discussion on this topic.

**Comment by JocelynDelalande** [comment]
Hey @diddledan :)

@diddledan @floogulinc would be more in favor of a one-time consistent adding of useful aliases (like you offered, to port from shuo), than alias-per-alias PR.

So not in favor for this PR alone :-1:  (for different reasons than @astorije  though)

**Comment by JocelynDelalande** [comment]
(with two -1, I close this PR)
